ContactList.cs

```diff
@@ -0,0 +1,60 @@
+﻿using HwCreateGame;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class ContactList
+{
+    private List<Contact> contacts;
+
+    public ContactList()
+    {
+        contacts = new List<Contact>();
+    }
+
+    // Добавление контакта
+    public void AddContact(Contact contact)
+    {
+        contacts.Add(contact);
+    }
+
+    // Поиск контактов по параметру
+    public List<Contact> SearchContacts(string parameter)
+    {
+        return contacts.Where(c => c.Name.Contains(parameter) || c.Surname.Contains(parameter) || c.Phone.Contains(parameter)).ToList();
+    }
+
+    // Отображение всех контактов
+    public void DisplayAllContacts()
+    {
+        Console.WriteLine("Список контактов:");
```

Comment:
What console is doing in the service class? In the task there is a mention: if you want to print to the console, it has to be handled separately in class that manages output to the console